dashboard/src/components/AppView/OtherResourcesTable/OtherResourcesTable.tsx

```diff
@@ -24,12 +24,21 @@ class OtherResourcesTable extends React.Component<IAppDetailsProps> {
     if (otherResources.length > 0) {
       otherResourcesSection = (
         <table>
+          <thead>
+            <tr>
+              <th>KIND</th>
+              <th>NAMESPACE</th>
+              <th>NAME</th>
+            </tr>
+          </thead>
           <tbody>
             {otherResources.map((r: IResource) => {
               return (
                 <tr key={`otherResources/${r.kind}/${r.metadata.name}`}>
                   <td>{r.kind}</td>
-                  <td>{r.metadata.namespace}</td>
+                  <td>
+                    {r.metadata.namespace || <i style={{ color: "lightgray" }}>Not Namespaced</i>}
```

Comment:
this isn't really related to this PR right?

Reply:
The PR description contain two points, one for fixing the `List` handling and the second to fix how `otherResources` are represented. I did that in just one PR because I realize that "Other resources" were not being properly represented while fixing the first point. Since it was small enough I didn't see the need to split in two different PRs. Anyway, if you find this conflictive I can split this in two PR.